test/sync.js

```diff
@@ -170,35 +170,19 @@ tape('sync: syncfile replication: osm-p2p-syncfile', function (t) {
   })
 })
 
-tape('sync: media: desktop <-> desktop', function (t) {
-  t.plan(18)
-
-  function writeBlob (api, filename, cb) {
-    var pending = 3
-    var ws = api.media.createWriteStream('original/' + filename, done)
-    ws.end(filename)
-
-    ws = api.media.createWriteStream('preview/' + filename, done)
-    ws.end(filename)
-
-    ws = api.media.createWriteStream('thumbnail/' + filename, done)
-    ws.end(filename)
-
-    function done (err) {
-      t.error(err)
-      if (!--pending) cb()
-    }
-  }
+tape.only('sync: big media data: desktop <-> desktop', function (t) {
+  t.plan(12)
 
   var opts = {api1:{deviceType:'desktop'}, api2:{deviceType:'desktop'}}
   createApis(opts, function (api1, api2, close) {
     var pending = 4
+    var total = 200
 
     api1.sync.listen()
     api1.sync.on('target', written.bind(null, null))
     api2.sync.listen()
     api2.sync.on('target', written.bind(null, null))
-    writeBlob(api1, 'hello_world.png', written)
+    helpers.writeBigData(api1, total, written.bind(null, null))
```

Comment:
Why bind `null` to the `err` slot? It looks like `writeBigData` will return an error.

Reply:
good point! was just a copy/paste